src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +179,36 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssuesCount() {
+        return (int) getElementsList().stream().filter((element) -> !element.getIssue().isOpen()).count();
+    }
```

Comment:
Yes, that kind of UI-data separation can be useful. We should do that at  some point.